caput/scripts/runner.py

```diff
@@ -314,9 +323,9 @@ def queue(configfile, submit=False, lint=True):
     rconf["workdir"] = workdir
     rconf["finaldir"] = finaldir
     rconf["scriptpath"] = fixpath(__file__)
-    rconf["logpath"] = join(jobdir, "jobout.log")
-    rconf["configpath"] = join(jobdir, "config.yaml")
-    rconf["statuspath"] = join(jobdir, "STATUS")
+    rconf["logpath"] = PurePath.joinpath(jobdir, "jobout.log")
+    rconf["configpath"] = PurePath.joinpath(jobdir, "config.yaml")
+    rconf["statuspath"] = PurePath.joinpath(jobdir, "STATUS")
```

Comment:
Presumably when the formatting is done below, these Path objects get converted into strings when the `script % rconf` is evaulated.
```suggestion
    rconf["logpath"] = jobdir / "jobout.log"
    rconf["configpath"] = jobdir / "config.yaml"
    rconf["statuspath"] = jobdir / "STATUS"
```

Reply:
Yes, I tried that, they get converted as expected.